test.js

```diff
@@ -315,6 +315,34 @@ test('align option `left`', t => {
 	`);
 });
 
+test('align option (left) does not throw when colorized content > columns', t => {
+	console.log('process.stdout.columns', process.stdout.columns);
+	const longContent = chalk.green('ab').repeat(process.stdout.columns);
+	t.notThrows(() => {
+		boxen(longContent, {
+			align: 'left'
+		});
+	});
+});
+
+test('align option (center) does not throw when colorized content > columns', t => {
+	const longContent = chalk.green('ab').repeat(process.stdout.columns);
+	t.notThrows(() => {
+		boxen(longContent, {
+			align: 'center'
+		});
+	});
+});
+
+test('align option (right) does not throw when colorized content > columns', t => {
+	const longContent = chalk.green('ab').repeat(process.stdout.columns);
+	t.notThrows(() => {
+		boxen(longContent, {
+			align: 'right'
+		});
+	});
+});
```

Comment:
I know that those are not the best tests as they don't quite test the final result but I won't have time to dig into writing better ones.